frontends/verilog/verilog_parser.y

```diff
@@ -1955,6 +1955,7 @@ assert:
 			delete $5;
 		} else {
 			AstNode *node = new AstNode(assume_asserts_mode ? AST_ASSUME : AST_ASSERT, $5);
+			SET_AST_NODE_LOC(node, @1, @7);
```

Comment:
```suggestion
			SET_AST_NODE_LOC(node, @1, @6);
```
The trailing semicolon probably should not be counted. The same goes for the following rules.

Reply:
Corrected